content/00.build/30.zksync-sso/50.getting-started.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Getting Started
+description: Learn how to integrate ZKsync SSO into your app.
+---
+
+Integrating ZKsync SSO into your app requires minimal amount of changes.
+::callout{icon="i-heroicons-light-bulb"}
+ZKsync SSO can be used standalone or added as one of the options in the onboarding library of your choice.
+::
+
+## Quick Start
+
+Just 2 steps:
+
+1. Install the ZKsync SSO SDK package.
+
+  ::code-group
+
+    ```bash [npm]
+    npm i zksync-account
+    ```
+
+    ```bash [yarn]
+    yarn add zksync-account
+    ```
+
+    ```bash [pnpm]
+    pnpm add zksync-account
+    ```
+
+    ```bash [bun]
+    bun add zksync-account
+    ```
+
+  ::
+
+1. Add ZKsync SSO connector to your app.
+
+::code-group
+
+  ```ts [wagmi]
+  import { zksync } from "viem/chains";
+  import { createConfig, connect } from "@wagmi/core";
+  import { zksyncAccountConnector } from "zksync-account/connector";
+
+  const ssoConnector = zksyncAccountConnector({
+    // Optional session configuration
+    session: {
+      feeLimit: parseEther("0.1"),
+      // Allow transfers to a specific address with a limit of 0.1 ETH
+      transferPolicies: [
```

Comment:
We should include call policies example also

Reply:
@JackHamer09  - are you able to add this?